### PR TITLE
Fixes the URL to Doctrine ORM events docs

### PR DESCRIPTION
**Pull request by IckleChris**

### CODE REVIEW

File: core/events.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ feature](http://symfony.com/doc/current/components/dependency_injection/autowiri
 
 Alternatively, [the subscriber must be registered manually](http://symfony.com/doc/current/components/http_kernel/introduction.html#creating-an-event-listener).
 
-Doctrine events ([ORM](http://doctrine-orm.readthedocs.org/en/latest/reference/events.html#reference-events-lifecycle-events), [MongoDB ODM](https://www.doctrine-project.org/projects/doctrine-mongodb-odm/en/latest/reference/events.html#lifecycle-events))
+Doctrine events ([ORM](https://www.doctrine-project.org/projects/doctrine-orm/en/latest/reference/events.html), [MongoDB ODM](https://www.doctrine-project.org/projects/doctrine-mongodb-odm/en/latest/reference/events.html#lifecycle-events))
```
Comment (alanpoulain) [inline core/events.md]:
```suggestion
Doctrine events ([ORM](https://www.doctrine-project.org/projects/doctrine-orm/en/current/reference/events.html), [MongoDB ODM](https://www.doctrine-project.org/projects/doctrine-mongodb-odm/en/latest/reference/events.html#lifecycle-events))
```

Comment (IckleChris) [reply core/events.md]: Yeah I wasn't sure what version is best to use, I just retained `latest` from the previous version.

Comment (alanpoulain) [reply core/events.md]: Yes, but since there is the new 3.0 version, I think it's best to change it to current.

Comment (IckleChris) [reply core/events.md]: Yep agreed.

**Comment by alanpoulain** [comment]
Thanks @IckleChris.

**Pull request merged**